CONTRIBUTING.md

```diff
@@ -130,12 +130,14 @@ possible.
 ### Creating a module
 
 1.  Run `yarn create-module`
-2.  Enter in a module name, description and author
+2.  Enter in a module name, description and author. You'll be prompted to choose if you also want to
+    create a CSS module.
 3.  (optional) Add any required dependencies on other modules
 4.  (optional) If you added any extra dependencies, run `yarn`
 5.  Start Storybook `yarn start`
 6.  Navigate to [http://localhost:9001/](http://localhost:9001/) and find your new module's story
-7.  Begin editing your new React component in `modules/canvas-kit-react-<NAME>/index.js`!
+7.  Begin editing your new React component in `modules/canvas-kit-react-<NAME>/react/index.ts` and
```

Comment:
These are still the old file paths. They should be `modules/<MODULE_NAME>/react` AND `modules/<MODULE_NAME>/css` respectively.

(open to other suggestions other than `MODULE_NAME`)